test/services/role/services/userPermissions.test.js

```diff
@@ -0,0 +1,94 @@
+const { expect } = require('chai');
+const app = require('../../../../src/app');
+const testObjects = require('../../helpers/testObjects')(app);
+
+const userPermissions = app.service('/permissions/user');
+
+describe('userPermissions', async () => {
+	const ROLES = {
+		TEST: 'test',
+		OTHER: 'other',
+		EXTENDED: 'extended',
+		MULTIPLE: 'multiple',
+		XX: 'xx',
+		NOTHING: 'nothing spezial',
+  };
+  const testPermissions = [
+		'SINGING',
+		'DANCE_RAIN',
+		'WALK_LINES',
+		'RUN_FLOOR',
+  ];
+  
+  	const otherPermissions = [
+		'SITTING',
+		'SITTING_ON_CHAIR',
+		'SITTING_ON_DESK',
+	];
+  
+	before(async () => {
+		const testRole = await testObjects.createTestRole({
+			name: ROLES.TEST,
+			permissions: testPermissions,
+    });
+    
+    const testRoleWithDiffrentPermissons = await testObjects.createTestRole({
```

Comment:
Codacy found an issue: [Expected indentation of 2 tabs but found 4 spaces.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5786177)